platform.md

```diff
@@ -328,7 +330,7 @@ The lifecycle:
 - SHALL write the resolved build plan from the detected group to `<plan>`
 
 #### `analyzer`
```

Comment:
To resolve run image mixins, analyzer will need `-run-image` and `-stack` (provides the fallback run images) inputs.

Reply:
Addressed in 015fe3d of #172